### PR TITLE
Check import orders

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [MESSAGE CONTROL]
-disable=missing-docstring,redefined-outer-name
+disable = missing-docstring,redefined-outer-name
```
Comment (thewilkybarkid) [inline .pylintrc]: Just matching INI styles elsewhere.

File: profiles/api/oauth2.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 from urllib.parse import urlencode
+
+from flask import Blueprint, current_app, json, jsonify, make_response, redirect, request, url_for
 import requests
-from flask import Blueprint, current_app, make_response, redirect, request, jsonify, url_for, json
-from profiles.api.errors import InvalidClient, InvalidRequest, UnsupportedGrantType, InvalidGrant, \
-    ClientInvalidRequest, ClientUnsupportedResourceType, ClientInvalidScope
-from profiles.utilities import remove_none_values
 from werkzeug.exceptions import BadRequest
 from werkzeug.wrappers import Response
```
Comment (giorgiosironi) [inline profiles/api/oauth2.py]: agreed, application code imported after libraries

Comment (giorgiosironi) [reply profiles/api/oauth2.py]: (and the order of imports in the same line ofc)

**Comment by thewilkybarkid** [comment]
This doesn't mandate a line between system/libraries and libraries/application imports. Possible?

**Pull request merged**